carmajava/launch/dev_config/saxton_cav.urdf

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<!--
+  Copyright (C) 2018 LEIDOS.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<!-- Green Cadillac Robot Description File

```

Comment:
This whole comment section probably needs to be updated to reflect current reality.  It's used on all Cadillacs now, right?  If you agree, please write up a Github issue to update all of the applicable urdf files.

Reply:
Yeah, it's used on all the cadillacs, so the language could use an update for sure. I think the data is still correct though.